dts/bindings/clock/nuvoton,npcx-pcc.yaml

```diff
@@ -11,6 +11,13 @@ properties:
     reg:
         required: true
 
+    ram-pd-depth:
+        required: false
+        type: int
```

Comment:
Do you want to provide an enum as well, or is any bit depth really valid?

Reference on enums: https://docs.zephyrproject.org/latest/guides/dts/bindings.html#enum-values

Reply:
Done.